src/MoBi.Presentation/Presenter/ParameterValuesPresenter.cs

```diff
@@ -80,5 +92,22 @@ public void UpdateDimension(ParameterValueDTO valueObject, IDimension newDimensi
 
          AddCommand(macroCommand);
       }
+
+      public void ShowContextMenu(IViewItem objectRequestingPopup, Point popupLocation) =>
+         _viewItemContextMenuFactory.CreateFor(objectRequestingPopup, this).Show(_view, popupLocation);
+
+      public void AddNewParameterValue()
+      {
+         _modalPresenter.Text = AppConstants.Captions.SelectLocalReferencePoint;
+         //The next 2 lines should not be changed, as they are used to encapsulate the presenter
```

Comment:
can you just state that the 'order of the next two lines' should not be changed instead of the 'lines should not be changed'